<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
  <title>AAAI 2021<br>Artificial Intelligence for Drug Discovery | Artificial Intelligence for Drug Discovery, AAAI 2021</title>
  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <link rel="icon" type="image/png" href="/images/favicon-32x32.svg">
  <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;700&display=swap" rel="stylesheet">
  <link href="https://deepgraphlearning.github.io/DrugTutorial_AAAI2021/css/style.css" rel="stylesheet">
  
  <meta property="og:title" content="" />
  <meta property="og:type" content="" />
  <meta property="og:url" content="" />
  <meta property="og:image" content="" />
  <meta property="og:description" content="" />
</head>
<body class='page '>
  <div id="main-menu-mobile" class="main-menu-mobile">
  <ul>
    
  </ul>
</div>

  <div id="wrapper" class="wrapper">
    
    <div class="container pb-6 pt-6 pt-md-10 pb-md-10">
  <div class="row justify-content-start">
    <div class="col-12 col-md-12">
      <h1 class="title">AAAI 2021<br>Artificial Intelligence for Drug Discovery</h1>
      
<div class="row">
  
  <div class="col-12 col-md-4 mb-2">
    <div class="team team-summary team-summary-large">
      
      <div class="team-image">
        <img alt="Jian Tang" class="img-fluid mb-2" src="https://deepgraphlearning.github.io/DrugTutorial_AAAI2021/images/team/jian_tang.jpg" />
      </div>
      
      <div class="team-meta">
        <h2 class="team-name"><a href="https://jian-tang.com/">Jian Tang</a></h2>
        <p class="team-description">Assistant Professor<br />Mila - Quebec AI Institute</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-4 mb-2">
    <div class="team team-summary team-summary-large">
      
      <div class="team-image">
        <img alt="Fei Wang" class="img-fluid mb-2" src="https://deepgraphlearning.github.io/DrugTutorial_AAAI2021/images/team/fei_wang.jpeg" />
      </div>
      
      <div class="team-meta">
        <h2 class="team-name"><a href="https://wcm-wanglab.github.io/index.html">Fei Wang</a></h2>
        <p class="team-description">Associate Professor<br />Cornell University</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-4 mb-2">
    <div class="team team-summary team-summary-large">
      
      <div class="team-image">
        <img alt="Feixiong Cheng" class="img-fluid mb-2" src="https://deepgraphlearning.github.io/DrugTutorial_AAAI2021/images/team/feixiong_cheng.jpeg" />
      </div>
      
      <div class="team-meta">
        <h2 class="team-name"><a href="https://www.lerner.ccf.org/gmi/cheng/">Feixiong Cheng</a></h2>
        <p class="team-description">Principal Investigator<br />Lerner Research Institute</p>
      </div>
    </div>
  </div>
  
</div>

<h2 id="abstract">Abstract</h2>
<p>Drug discovery is a long and costly process, taking on average 10 years and 2.5 billion dollars to develop a new drug.
Artificial intelligence has the potential to significantly accelerate the process of drug discovery by analyzing a large amount of data generated in the biomedical domain such as bioassays, chemical experiments, and biomedical literature.
Recently, there is a growing interesting in developing AI techniques for drug discovery in many different communities including machine learning, data mining, and biomedical community. In this tutorial, we will provide a detailed introduction to key problems in drug discovery such as molecular property prediction, de novo molecular design and molecular optimization, retrosynthesis reaction and prediction, and drug repurposing and combination, and also key technique advancements with artificial intelligence for these problems. This tutorial can be served as introduction materials for both computer scientist interested in drug discovery as well as drug discovery practitioners for learning the latest AI techniques along this direction.</p>

<h2 id="outline">Outline</h2>
<p>The proposed tutorial would be 3.5 hours in length. 
An outline of the tutorial (including an approximate breakdown by presenter) is as follows:</p>

<ul>
  <li>Drug Discovery Overview [20 min, presented by Feixiong]</li>
  <li>Deep Learning, Traditional Network-based Methods, Graph Representation Learning [15 min, presented by Fei]
    <ul>
      <li>Convolutional Neural Networks and Recurrent Neural Networks</li>
      <li>Graph Convolutional Networks [GCN <a class="citation" href="#kipf2016semi">(Kipf &amp; Welling, 2016)</a>, MPNN <a class="citation" href="#gilmer2017neural">(Gilmer et al., 2017)</a>, GIN<a class="citation" href="#xu2018powerful">(Xu et al., 2018)</a>]</li>
    </ul>
  </li>
  <li>Molecule Properties Prediction [30 min, presented by Jian]
    <ul>
      <li>Variational autoencoder-based approach <a class="citation" href="#jin2018junction">(Jin et al., 2018)</a></li>
      <li>Autoregressive methods [GCPN <a class="citation" href="#you2018graph">(You et al., 2018)</a>]</li>
      <li>Normalizing Flow-based approaches [GraphAF <a class="citation" href="#shi2020graphaf">(Shi et al., 2020)</a>, MoFlow <a class="citation" href="#zang2020moflow">(Zang &amp; Wang, 2020)</a>]</li>
    </ul>
  </li>
  <li>De novo Molecule Generation and Optimization. [40 min, presented by Jian and Fei]
    <ul>
      <li>Variational autoencoder-based approach [JTVAE <a class="citation" href="#jin2018junction">(Jin et al., 2018)</a>]</li>
      <li>Autoregressive methods [GCPN <a class="citation" href="#you2018graph">(You et al., 2018)</a>]</li>
      <li>Normalizing Flow-based approaches [GraphAF <a class="citation" href="#shi2020graphaf">(Shi et al., 2020)</a>, MoFlow <a class="citation" href="#zang2020moflow">(Zang &amp; Wang, 2020)</a>]</li>
    </ul>
  </li>
  <li>Reaction Prediction and Retrosynthesis. [30 min, presented by Jian]
    <ul>
      <li>Reaction prediction <a class="citation" href="#jin2017predicting">(Jin et al., 2017)</a> <a class="citation" href="#schwaller2019molecular">(Schwaller et al., 2019)</a> <a class="citation" href="#sacha2020molecule">(Sacha et al., 2020)</a>
        <ul>
          <li>Retrosynthesis <a class="citation" href="#dai2019retrosynthesis">(Dai et al., 2019)</a> <a class="citation" href="#shhi2020a">(Shi et al., 2020)</a> <a class="citation" href="#sacha2020molecule">(Sacha et al., 2020)</a></li>
        </ul>
      </li>
    </ul>
  </li>
  <li>Multiomics and Clinical Data-based Drug Repurposing. [45 min, presented by Feixiong and Fei]</li>
  <li>Other Topics. [15 min, presented by Fei]</li>
  <li>Conclusion and Future Directions. [15 min, presented by Fei]</li>
</ul>

<h2 id="organizers">Organizers</h2>

<ul>
  <li>Jian Tang, <a href="https://jian-tang.com/">website</a><br />
    <ul>
      <li><strong>Brief biograph.</strong>  Jian Tang is currently an Assistant Professor at Montr'eal Institute for Learning Algorithms (Mila)—a research institute focusing on deep learning and reinforcement learning led by Turing Award Winner Yoshua Bengio—starting from December 2017. His research focuses on graph representation learning, graph neural networks, drug discovery, and knowledge graphs. He is named to the first cohort of Canada CIFAR Artificial Intelligence Chairs (CIFAR AI Research Chair). He was a research fellow in University of Michigan and Carnegie Mellon University. He was a researcher in Microsoft Research Asia for two years. He received the best paper award of ICML’14 and nominated for the best paper of WWW’16. Most of his papers are published in top-tier venues across artificial intelligence, machine learning and data mining conferences (ICML, NeurIPS, ICLR, AAAI, IJCAI, KDD, WWW, and WSDM). He co-organized a tutorial on graph representation learning at KDD 2017 and AAAI 2019, organized a few workshops on graph representation learning at SDM 2019, CIKM 2019, AAAI 2020, and ICML 2020. He published quite a few representative work on graph representation learning (including LINE, LargeVis, RotatE) and recently has been very actively working on graph representation learning for drug discovery.</li>
      <li><strong>Relevant reviewing experience.</strong>  Jian Tang has served as a reviewer at the major conferences of machine learning, data mining, and natural language processing communities including NIPS, ICML, ICLR, AAAI, IJCAI, ACL, EMNLP, KDD, WWW, and WSDM.</li>
    </ul>
  </li>
  <li>Fei Wang, <a href="https://wcm-wanglab.github.io/index.html">website</a><br />
    <ul>
      <li><strong>Brief biograph.</strong>  Fei Wang is currently an Associate Professor of Health Informatics in Department of Population Health Sciences, Weill Cornell Medicine, Cornell University. His major research interest is data mining and its applications in health data science. He has published more than 250 papers in AI and medicine, which have received more than 12.7K citations <a href="https://scholar.google.com/citations?hl=en&amp;user=FjCbjDYAAAAJ">Google Scholar</a>. His H-index is 56. His papers have won 8 best paper awards at top international conferences on data mining and medical informatics. His team won the championship of the NIPS/Kaggle Challenge on Classification of Clinically Actionable Genetic Mutations in 2017 and Parkinson’s Progression Markers’ Initiative data challenge organized by Michael J. Fox Foundation in 2016. Dr. Wang is the recipient of the NSF CAREER Award in 2018, the inaugural research leadership award in IEEE International Conference on Health Informatics (ICHI) 2019. Dr. Wang is the chair of the Knowledge Discovery and Data Mining working group in American Medical Informatics Association (AMIA).</li>
      <li><strong>Relevant reviewing experience.</strong>  Fei Wang has served as senior program committee member/area chair of conferences including AAAI, IJCAI, KDD, CIKM, ICDM, SDM. He also reviews major medical and interdisciplinary journals including Nature Medicine, Nature Communications, Annals of Internal Medicine, etc.</li>
    </ul>
  </li>
  <li>Feixiong Cheng, <a href="https://www.lerner.ccf.org/gmi/cheng/">website</a><br />
    <ul>
      <li><strong>Brief biograph.</strong>  Feixiong Cheng, PhD, is a principal investigator with Cleveland Clinic’s Genomic Medicine Institute. Dr. Cheng is working to develop computational and experimental network medicine technologies for advancing the characterization of disease heterogeneity, thereby approaching the goal of coordinated, patient-centered strategies to innovative diagnostics and therapeutics development. The primary goal of Dr. Cheng’s lab is to combine tools from genomics, network medicine, bioinformatics, computational biology, chemical biology, and experimental pharmacology and systems biology assays (e.g., single cell sequencing and iPS-derived cardiomyocytes), to address the challenging questions toward understanding of various human complex diseases (e.g., cardio-oncology, pulmonary vascular diseases, and Alzheimer’s disease), which could have a major impact in identifying novel real-world data-driven diagnostic biomarkers and therapeutic targets for precision medicine. From 2013 to 2017, Dr. Cheng was trained as Postdoctoral Research Fellow in the field of pharmacogenomics and network medicine across Vanderbilt University Medical Center, Northeastern University, and Dana-Farber Cancer Institute. During 2017-2018, Dr. Cheng was promoted to Research Assistant Professor working with two of the world’s leading experts in the field of network medicine, Drs. Albert-Laszlo Barabasi and Joseph Loscalzo, with dual appointment at Northeastern University and Harvard Medical School. Dr. Cheng has received several awards, including NIH Pathway to Independence Award (K99/R00), SCI highly cited papers reward, and Vanderbilt Postdoc of the Year Honorable mention.</li>
    </ul>
  </li>
</ul>

<h2 id="references">References</h2>
<ol class="bibliography"><li><span id="sun2019infograph">Sun, F.-Y., Hoffmann, J., Verma, V., &amp; Tang, J. (2020). Infograph: Unsupervised and semi-supervised graph-level representation learning via mutual information maximization. <i>ICLR</i>.</span></li>
<li><span id="shi2020graphaf">Shi, C., Xu, M., Zhu, Z., Zhang, W., Zhang, M., &amp; Tang, J. (2020). GraphAF: a flow-based autoregressive model for molecular graph generation. <i>ICLR</i>.</span></li>
<li><span id="shhi2020a">Shi, C., Xu, M., Guo, H., Zhang, M., &amp; Tang, J. (2020). A Graph to Graphs Framework for Retrosynthesis Prediction. <i>ICML</i>.</span></li>
<li><span id="gottipati2020learning">Gottipati, S. K., Sattarov, B., Niu, S., Pathak, Y., Wei, H., Liu, S., Thomas, K. M. J., Blackburn, S., Coley, C. W., Tang, J., &amp; others. (2020). Learning To Navigate The Synthetically Accessible Chemical Space Using Reinforcement Learning. <i>ICML</i>.</span></li>
<li><span id="jin2018junction">Jin, W., Barzilay, R., &amp; Jaakkola, T. (2018). Junction tree variational autoencoder for molecular graph generation. <i>ICML</i>.</span></li>
<li><span id="you2018graph">You, J., Liu, B., Ying, Z., Pande, V., &amp; Leskovec, J. (2018). Graph convolutional policy network for goal-directed molecular graph generation. <i>Advances in Neural Information Processing Systems</i>, 6410–6421.</span></li>
<li><span id="zang2020moflow">Zang, C., &amp; Wang, F. (2020). MoFlow: An Invertible Flow Model for Generating Molecular Graphs. <i>Proceedings of the 26th ACM SIGKDD International Conference on Knowledge Discovery &amp; Data Mining</i>, 617–626.</span></li>
<li><span id="sun2020graph">Sun, M., Zhao, S., Gilvary, C., Elemento, O., Zhou, J., &amp; Wang, F. (2020). Graph convolutional networks for computational drug development and discovery. <i>Briefings in Bioinformatics</i>, <i>21</i>(3), 919–935.</span></li>
<li><span id="hu2019strategies">Hu, W., Liu, B., Gomes, J., Zitnik, M., Liang, P., Pande, V., &amp; Leskovec, J. (2019). Strategies for Pre-training Graph Neural Networks. <i>ArXiv Preprint ArXiv:1905.12265</i>.</span></li>
<li><span id="gilmer2017neural">Gilmer, J., Schoenholz, S. S., Riley, P. F., Vinyals, O., &amp; Dahl, G. E. (2017). Neural message passing for quantum chemistry. <i>ArXiv Preprint ArXiv:1704.01212</i>.</span></li>
<li><span id="kipf2016semi">Kipf, T. N., &amp; Welling, M. (2016). Semi-supervised classification with graph convolutional networks. <i>ArXiv Preprint ArXiv:1609.02907</i>.</span></li>
<li><span id="xu2018powerful">Xu, K., Hu, W., Leskovec, J., &amp; Jegelka, S. (2018). How powerful are graph neural networks? <i>ArXiv Preprint ArXiv:1810.00826</i>.</span></li>
<li><span id="jin2017predicting">Jin, W., Coley, C., Barzilay, R., &amp; Jaakkola, T. (2017). Predicting organic reaction outcomes with weisfeiler-lehman network. <i>Advances in Neural Information Processing Systems</i>, 2607–2616.</span></li>
<li><span id="schwaller2019molecular">Schwaller, P., Laino, T., Gaudin, T., Bolgar, P., Hunter, C. A., Bekas, C., &amp; Lee, A. A. (2019). Molecular transformer: A model for uncertainty-calibrated chemical reaction prediction. <i>ACS Central Science</i>, <i>5</i>(9), 1572–1583.</span></li>
<li><span id="sacha2020molecule">Sacha, M., Błaż, M., Byrski, P., Włodarczyk-Pruszyński, P., &amp; Jastrzębski, S. (2020). Molecule Edit Graph Attention Network: Modeling Chemical Reactions as Sequences of Graph Edits. <i>ArXiv Preprint ArXiv:2006.15426</i>.</span></li>
<li><span id="dai2019retrosynthesis">Dai, H., Li, C., Coley, C., Dai, B., &amp; Song, L. (2019). Retrosynthesis prediction with conditional graph logic network. <i>Advances in Neural Information Processing Systems</i>, 8872–8882.</span></li>
<li><span id="zhou2020network">Zhou, Y., Hou, Y., Shen, J., Huang, Y., Martin, W., &amp; Cheng, F. (2020). Network-based drug repurposing for novel coronavirus 2019-nCoV/SARS-CoV-2. <i>Cell Discovery</i>, <i>6</i>(1), 1–18.</span></li>
<li><span id="zeng2019deepdr">Zeng, X., Zhu, S., Liu, X., Zhou, Y., Nussinov, R., &amp; Cheng, F. (2019). deepDR: a network-based deep learning approach to in silico drug repositioning. <i>Bioinformatics</i>, <i>35</i>(24), 5191–5198.</span></li>
<li><span id="zhou2020artificial">Zhou, Y., Wang, F., Jian, T., R., N., &amp; Cheng, F. (2020). Artificial Intelligence in Drug Repurposing. <i>The Lancet Digital Health</i>.</span></li>
<li><span id="chen2020idrug">Chen, H., Cheng, F., &amp; Li, J. (2020). iDrug: Integration of drug repositioning and drug-target prediction via cross-network embedding. <i>PLoS Computational Biology</i>, <i>16</i>(7), e1008040.</span></li>
<li><span id="cheng2019network">Cheng, F., Kovács, I. A., &amp; Barabási, A.-L. (2019). Network-based prediction of drug combinations. <i>Nature Communications</i>, <i>10</i>(1), 1–11.</span></li>
<li><span id="cheng2018network">Cheng, F., Desai, R. J., Handy, D. E., Wang, R., Schneeweiss, S., Barabási, A.-L., &amp; Loscalzo, J. (2018). Network-based approach to prediction and population-based validation of in silico drug repurposing. <i>Nature Communications</i>, <i>9</i>(1), 1–12.</span></li>
<li><span id="gysi2020network">Gysi, D. M., Valle, Í. D., Zitnik, M., Ameli, A., Gan, X., Varol, O., Sanchez, H., Baron, R. M., Ghiassian, D., Loscalzo, J., &amp; others. (2020). Network medicine framework for identifying drug repurposing opportunities for covid-19. <i>ArXiv Preprint ArXiv:2004.07229</i>.</span></li>
<li><span id="zhou2020networkb">Zhou, Y., Hou, Y., Shen, J., Kallianpur, A., Zein, J., Culver, D. A., Farha, S., Comhair, S., Fiocchi, C., Gack, M. U., &amp; others. (2020). A Network Medicine Approach to Investigation and Population-based Validation of Disease Manifestations and Drug Repurposing for COVID-19. <i>ChemRxiv</i>.</span></li></ol>

    </div>
  </div>
</div>
  </div>
  <div class="footer">
  <div class="container">
    <div class="row">
      <div class="col-12 mt-2 mb-4">
        <div class="footer-inner">
          <h3 class="footer-title">Artificial Intelligence for Drug Discovery, AAAI 2021</h3>
          <ul>
            
          </ul>
        </div>
      </div>
    </div>
  </div>
</div>
  <script type="text/javascript" src="https://deepgraphlearning.github.io/DrugTutorial_AAAI2021/js/scripts.js""></script>
  
</body>
</html>
